src/Kovan-DssSpell.t.sol

```diff
@@ -246,10 +246,10 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x8c38736FF2dA058a3aef53E59f39A42b34104636),        // populate with deployed spell if deployed
-            deployed_spell_created:         1628088292,        // use get-created-timestamp.sh if deployed
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
```

Comment:
Once the contract is deployed, we'll need to enter the address here and the timestamp on the next line, then re-test.